common/models/event.json

```diff
@@ -1,32 +1,39 @@
 {
-  "name": "Event",
-  "plural": "Events",
-  "base": "PersistedModel",
-  "idInjection": true,
-  "options": {
-    "validateUpsert": true
-  },
-  "properties": {
-    "name": {
-      "type": "string",
-      "required": true
+    "name": "Event",
```

Comment:
Do we want to keep the indentation as 2 spaces as that is consistent with what we have elsewhere?